applications/asset_tracker_v2/src/modules/Kconfig.data_module

```diff
@@ -31,30 +31,59 @@ config DATA_THREAD_STACK_SIZE
 	int "Data module thread stack size"
 	default 2560
 
-config GPS_BUFFER_MAX
-	int "GPS data ringbuffer entries"
+config DATA_GPS_BUFFER_COUNT
+	int "Number of GPS data ringbuffer entries"
+	range 1 100
```

Comment:
Please add a description on the first one, why the lower limit is 1 and the upper limit is 100.

Reply:
Will do.